docs/contributing/technical/setup-repo-with-git.md

```diff
@@ -55,16 +55,16 @@ upstream git@github.com:open-sauced/open-sauced.git (fetch)
 upstream git@github.com:open-sauced/open-sauced.git (push)
 ```
 
-### How to add a remote repository
+### How to Add a Remote Repository
 
-To pull in changes from the original repository, you will need to add a remote repository. To do this, run the following command:
+Adding a remote repository allows you to pull in changes from the original repository and keep your forked copy of the repository up to date.
+
+To add a remote repository, run the following command:
 
 ```shell
 git remote add upstream git@github.com:open-sauced/open-sauced.git
```

Comment:
- I'm not sure and need confirmation about this command.
   From the [official GitHub CLI docs](https://cli.github.com/manual/gh_repo_set-default) in the "Examples" section, they use this command: ` git remote add newrepo https://github.com/owner/repo`.
- We might want to update the remote URL example here because the `open-sauced`  repo has been archived.